core/blockly.js

```diff
@@ -125,6 +125,13 @@ Blockly.dragMode_ = Blockly.DRAG_NONE;
 Blockly.flyoutButtonCallbacks_ = {};
 
 /**
+ * Cached value for whether 3D is supported
+ * @type {boolean}
```

Comment:
I need to be able to tell the difference between has been checked and hasn't because is3dSupported check is expensive. It is cacheable though, thus the initialization to null as an indicator that we haven't run the check yet.